doc/import_export.rst

```diff
@@ -123,117 +88,62 @@ and their columns are a subset of names.
 	   capital_cost=1e5,
 	   p_max_pu=wind)
 
-Any attributes which are not specified will be given the default value from :doc:`components`.
-
 
 Adding components using pandas DataFrames
 =========================================
 
 To add multiple components whose static attributes are given in a
 pandas DataFrame, use
 
-``network.import_components_from_dataframe(dataframe, cls_name)``
-
-``dataframe`` is a pandas DataFrame whose index is the names of the
-components and whose columns are the non-default
-attributes. ``cls_name`` is the component name,
-e.g. ``"Line","Bus","Generator","StorageUnit``. If columns are missing
-then defaults are used. If extra columns are added, these are left in
-the resulting component DataFrame.
-
-.. code:: python
-
-    import pandas as pd
-
-    buses = ['Berlin', 'Frankfurt', 'Munich', 'Hamburg']
-
-    network.import_components_from_dataframe(pd.DataFrame({"v_nom" : 380,
-                                                           "control" : 'PV'},
-							  index=buses),
-					     "Bus")
-
-    network.import_components_from_dataframe(pd.DataFrame({"carrier" : "solar",
-                                                           "bus" : buses,
-							   "p_nom_extendable" : True,
-							   "capital_cost" : 6e4},
-							  index=[b+" PV" for b in buses]),
-					     "Generator")
+.. autofunction:: pypsa.io.import_components_from_dataframe
 
 To import time-varying information use
 
-``network.import_series_from_dataframe(dataframe, cls_name, attr)``
+.. autofunction:: pypsa.io.import_series_from_dataframe
 
-``cls_name`` is the component name, ``attr`` is the time-varying
-attribute and ``dataframe`` is a pandas DataFrame whose index is
-``network.snapshots`` and whose columns are a subset of the relevant
-components.
-
-Following the previous example:
-
-.. code:: python
-
-    import numpy as np
-
-    network.set_snapshots(range(10))
-    network.import_series_from_dataframe(pd.DataFrame(np.random.rand(10,4),
-                                                      columns=network.generators.index,
-						      index=range(10)),
-				         "Generator",
-					 "p_max_pu")
```

Comment:
Same as for `import_components_from_dataframe`:
- [ ] include example in docstring
- [ ] add note about `madd`